orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/TerminateInstancesTask.groovy

```diff
@@ -45,8 +47,14 @@ class TerminateInstancesTask extends AbstractCloudProviderAwareTask implements T
         "terminate.account.name": account,
         "terminate.region"      : stage.context.region,
         "kato.last.task.id"     : taskId,
-        "kato.task.id"          : taskId, // TODO retire this.
-        "terminate.instance.ids": stage.context.instanceIds,
+        "terminate.instances"   : stage.context.instances,
     ])
   }
+
+  static Map convert(Stage stage) {
+    def operation = [:] << stage.context
+    def instances = operation.remove("instances")
+    operation.instanceIds = instances.collect { it.id }
```

Comment:
I don't think we want to restrict to a single instance - in the AWS case the terminate call supports a list of instances and we shouldn't lose the ability to perform that batch operation

I would definitely rather see the orchestration across regions done up in orca than part of the clouddriver API call though